Configuration/StandardSequences/python/DigiToRaw_cff.py

```diff
@@ -31,6 +31,10 @@
 from Configuration.Eras.Modifier_phase2_hcal_cff import phase2_hcal
 phase2_hcal.toReplaceWith(DigiToRaw, DigiToRaw.copyAndExclude([hcalRawData]))
 
+# Remove siPixelRawData until we have phase1 pixel digis
+from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
+phase2_tracker.toReplaceWith(DigiToRaw, DigiToRaw.copyAndExclude([siPixelRawData,rpcpacker])) # FIXME
```

Comment:
(not for @ebrondol but for wider audience) I don't think removing `rpcpacker` should depend on `phase2_tracker`. Maybe `phase2_muon`? (assuming it really needs to be removed)